ros_message/src/time.rs

```diff
@@ -37,6 +38,28 @@ impl Time {
         Self::default()
     }
 
+    /// Get the current time.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use ros_message::Time;
+    /// # #[derive(Default)]
+    /// # struct Header {
+    /// #   stamp: Time,
+    /// # }
+    /// # #[derive(Default)]
+    /// # struct Message {
+    /// #   header: Header,
+    /// # }
+    /// # let mut message = Message::default();
+    /// message.header.stamp = Time::now();
+    /// ```
+    #[inline]
+    pub fn now() -> Self {
+        time::SystemTime::now().into()
```

Comment:
In `roscpp`, `ros::Time::now()` might returns the simulated time so this implementation is not correct I think.
And we already have `rosrust::now()`.
But something like `rosrust::wall_time::now()` using `std::time::SystemTime::now()` would be nice.

Reply:
Implemented your suggestion :)